esy-lib/Path.rei

```diff
@@ -20,6 +20,8 @@ let currentPath: unit => t;
 let homePath: unit => t;
 let dataPath: unit => t;
 
+let supportsLongPaths: unit => bool;
```

Comment:
@andreypopp - are you OK with the naming and location of this API? Some other options I considered were the `Environment` module or the `Fs` module. Let me know if you have a preference!

Reply:
I think `Environment` is more suitable for that.

Reply:
Thank you @andreypopp ! 👍